README.md

```diff
@@ -1,12 +1,17 @@
 Hautelook Templated URI Router
 ==============================
 
-Symfony2 UrlGenerator that provides a [RFC-6570][RFC-6570] compatible router and URL Generator.
-Currently it is extremely naive, and incomplete.
-However, it does what we need it to do. Contributions are welcome.
+Symfony URL generator that provides an [RFC-6570](https://tools.ietf.org/html/rfc6570)-compatible router.
+
+Currently, it is extremely naive and incomplete. However, it does what we need it to do. Contributions are welcome.
 
 [![Build Status](https://secure.travis-ci.org/hautelook/TemplatedUriRouter.png?branch=master)](https://travis-ci.org/hautelook/TemplatedUriRouter)
 
+## Requirements
```

Comment:
please remove this section, requirements can be seen in the composer.json

Reply:
Done.